server/services/alertsPageService.ts

```diff
@@ -53,9 +54,9 @@ export default class AlertsPageService {
   private mapToApiParams(queryParams: PagedListQueryParams) {
     const apiParams = { ...queryParams }
 
-    apiParams.from = apiParams.from && formatDateISO(parseDate(apiParams.from))
-    apiParams.to = apiParams.to && formatDateISO(parseDate(apiParams.to))
-    apiParams.page = apiParams.page && +apiParams.page - 1 // Change page to zero based for API query
+    if (apiParams.from) apiParams.from = apiParams.from && formatDateISO(parseDate(apiParams.from))
```

Comment:
with the `if (apiParams.from)` does that make the later part with the `apiParams.from &&` redundant?